demo_fastapi/sales.py

```diff
@@ -57,27 +73,118 @@ def create_order(
         )
 
     # if everything ok, make an order
-    order_id = uuid1().int
+    order_id = random.randrange(10**8)  # 12345  # uuid1().int
```

Comment:
same as above :-)

Reply:
Another question here: Can we be sure that this number is unique? 
If you prefer to use integer and do not want to use uuid1, you could also use [sonyflae](https://pypi.org/project/sonyflake-py/). The advantage of such numbers is that they are meaningful and carry some useful information.

Reply:
No, I don't think this would be necessarily unique for a bigger app with many orders. I only wanted to have a prettier number in the demo app because uuid1 returns huge not so pretty numbers.